bot/src/main.rs

```diff
@@ -741,6 +734,50 @@ fn get_config() -> BoxResult<(Config, Arc<RpcClient>, Box<dyn GenericStakePool>)
     )
     .unwrap();
 
+    let default_json_rpc_url = match cluster {
+        Cluster::MainnetBeta => "http://api.mainnet-beta.solana.com",
+        Cluster::Testnet => "http://api.testnet.solana.com",
+    }
+    .to_string();
+
+    // Create a list of RPC URLs to try. If a URL is specified on the command-line, try that first.
+    // If the URL specified on the command-line is not "healthy," or if no URL was specified on the
+    // command-line, fall back to the public solana.com RPC url.
+    let json_rpc_urls_to_try: Vec<String> = match value_t!(matches, "json_rpc_url", String) {
```

Comment:
does this work ATM?  I don't see the arg being configured to be accepted multiple times

Reply:
Yeah I'm pretty sure clap is going to reject a command line that specifies `--url URL` more than once unless [`Arg::multiple(true)`](https://docs.rs/clap/2.33.1/clap/struct.Arg.html#method.multiple) (rtfm, it's a little confusing :sweat_smile:) is set when defining `"json_rpc_url"`

Reply:
I think you're missing my point.  I'm saying that as configured, that `Vec<_>` will never see more than one item

Reply:
If it hits line 751 it will have two items in it. Otherwise, yeah, it will only have one item.

Reply:
I see.  I think I misunderstood the intention of the change set.  I'm not sure how valuable a hard coded fallback might be.  It seems like it'd be more flexible to accept all potential RPC endpoints on the command line.

If this is an improvement, fine by me

Reply:
Yeah, if I had to do it again that's how I'd do it. This works and I have other things on my plate, so I think I'll ask @c3pko to merge.